test/int.jl

```diff
@@ -339,8 +339,8 @@ let i = MyInt26779(1)
 end
 
 @testset "rounding division" begin
-    for x = -100:100
-        for y = 1:100
+    for x = -50:50
```

Comment:
why change these existing tests

Reply:
Probably there is no need so reverting these changes

Reply:
Probably there is no need, so reverting this change